test/remote/gateways/remote_ebanx_test.rb

```diff
@@ -187,7 +195,8 @@ def test_partial_refund
   def test_failed_refund
     response = @gateway.refund(@amount, '')
     assert_failure response
-    assert_match('Parameter hash not informed', response.message)
+    assert_equal 'Parameters hash or merchant_payment_code not informed', response.message
+    assert_equal 'BP-REF-1', response.error_code
```

Comment:
The error description for error BP-REF-1 changed. The [docs](https://developer.ebanx.com/docs/resources/errorCodes/) appear to be outdated.